apps/dialtone-documentation/docs/components/chip.md

```diff
@@ -9,16 +9,48 @@ figma_url: https://www.figma.com/design/W58r5BkO8qTw3vem9YieJd/DT9-Component-Lib
 ---
 
 <code-well-header>
-  <example-chip label="Chip" with-avatar/>
+  <dt-stack direction="row" gap="400">
+    <example-chip size="sm" label="Chip" hide-close-btn :interactive="false"/>
+    <example-chip size="sm" label="Chip" :interactive="false"/>
+    <example-chip size="sm" label="Chip" with-avatar/>
+    <example-chip size="sm" label="Chip" with-icon/>
+  </dt-stack>
 </code-well-header>
 
 <!-- <component-combinator component-name="DtChip" /> -->
 
 ## Variants and Examples
 
-### Base
+### Non Interactive
+
+Add `:interactive="false"` to make it a read-only, non-interactive Chip. This changes it from a `<button>` to a non-interactive, read-only Chip with no events or hover/active state. Note that this does not effect the interactivity of its `×` remove button.
```

Comment:
I wish we had made `interactive="false"` the default. We're seeing Chip used everywhere that aren't actually interactive but defaults to that. So I moved this up in the doc to increase visibility.

Reply:
Maybe we can make this the default in dialtone 10